app/views/planning_applications/assessment/tasks/index.html.erb

```diff
@@ -29,10 +29,16 @@
             }
           ) %>
 
-      <%= render "additional_services" %>
+      <% if @planning_application.pre_application? %>
+        <%= render "additional_services" %>
+      <% end %>
 
       <%= render "assessment_information" %>
 
+      <% if @planning_application.possibly_immune?  && !@planning_application.pre_application? %>
```

Comment:
Not very important but I wonder if we should make `possibly_immune?` return false for pre-applications, instead of remembering to use both checks?